skimage/registration/_optical_flow.py

```diff
@@ -76,7 +76,7 @@ def _tvl1(reference_image, moving_image, flow0, attachment, tightness,
                                              [1] + reference_image.ndim * [3])
 
         image1_warp = warp(moving_image, get_warp_points(grid, flow_current),
-                           mode='nearest')
```

Comment:
:eyes: Was this a bug?

Reply:
I think it was an unintentional use of the scipy.ndimage name rather than letting the function convert from the numpy.pad equivalent name ('edge'). The docstring to `warp` conforms to `numpy.pad` style names (i.e. 'edge' rather than its `scipy.ndimage` equivalent: 'nearest'). It just so happened that warp didn't raise an error on unrecognized modes and just passed them along unmodified.

Reply:
May be something to investigate :confused:...